addons/web/static/src/views/fields/properties/kanban_properties_field.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="web.KanbanPropertiesField" owl="1">
+        <div t-ref="properties" class="w-100">
+            <div class="o_kanban_property_field d-inline-flex flex-column justify-content-center align-items-start w-100 my-1"
+                t-foreach="propertiesList"
+                t-as="propertyConfiguration"
+                t-key="propertyConfiguration.name"
+                t-if="propertyConfiguration.value &amp;&amp; propertyConfiguration.view_in_kanban">
+                <div class="mw-100 text-truncate">
+                    <PropertyValue
+                        readonly="props.readonly"
```

Comment:
Linting: would go for alphabetical ordering, easier to find something when we search it in a list :)

Reply:
ok :) I just keep event handler at the end (to group by "normal attributes" vs "event handlers")